src/Symfony/Component/HttpFoundation/Session/Storage/MockFileSessionStorage.php

```diff
@@ -103,7 +103,7 @@ public function save()
 
         try {
             if ($data) {
-                file_put_contents($this->getFilePath(), serialize($data));
+                file_put_contents($this->getFilePath(), serialize($data), \LOCK_EX);
```

Comment:
Do you know if the lock is blocking? or we should handle the failure/retry?

Reply:
yes, this is a blocking lock